lms/templates/instructor/instructor_dashboard_2/cohort-form.underscore

```diff
@@ -95,7 +95,7 @@
                                             <%=
                                             interpolate(
                                                 // Translators: Any text between %(screen_reader_start)s and %(screen_reader_end)s is only read by screen readers and never shown in the browser.
-                                                '%(screen_reader_start)sWarning:%(screen_reader_end)s The previously selected content group was deleted. Select another content group.',
+                                                gettext('%(screen_reader_start)sWarning:%(screen_reader_end)s The previously selected content group was deleted. Select another content group.'),
```

Comment:
Thanks Andy! I'm just going through and reminding everyone. I would like to
get this tooling fixed eventually but until then, we play the hand we're
dealt.

On Wed, Sep 23, 2015 at 11:55 PM, Andy Armstrong <notifications@github.com>
wrote:

> In lms/templates/instructor/instructor_dashboard_2/cohort-form.underscore
> <https://github.com/edx/edx-platform/pull/9868#discussion_r40283656>:
>
> > @@ -95,7 +95,7 @@
> >                                              <%=
> >                                              interpolate(
> >                                                  // Translators: Any text between %(screen_reader_start)s and %(screen_reader_end)s is only read by screen readers and never shown in the browser.
> > -                                                '%(screen_reader_start)sWarning:%(screen_reader_end)s The previously selected content group was deleted. Select another content group.',
> > +                                                gettext('%(screen_reader_start)sWarning:%(screen_reader_end)s The previously selected content group was deleted. Select another content group.'),
>
> @sarina <https://github.com/sarina> My bad on this one. T&L had a retro
> about our i18n practices and and we now have a PR check list that should
> ensure that this doesn't happen again.
>
> See https://openedx.atlassian.net/wiki/display/TNL/August+27,+2015 for
> details.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/9868/files#r40283656>.
>